transitions/registration.js

```diff
@@ -272,5 +252,43 @@ module.exports = {
                 callback();
             }
         });
+    },
+    addPatient: function(options, callback) {
+        var doc = options.doc,
+            db = options.db,
+            audit = options.audit,
+            id = doc.patient_id,
+            patientNameField = (options.params && options.params.length && options.params[0]) || 'patient_name';
+
+        utils.getRegistrations({
+            db: db,
+            id: id
+        }, function(err, registrations) {
+            if (err) {
+                return callback(err);
+            }
+            if (registrations.length) {
+                // patient already registered, no action required
+                return callback();
+            }
+            db.medic.view('medic-client', 'people_by_phone', {
+                key: [ doc.from ],
+                include_docs: true
+            }, function(err, result) {
+                if (err) {
+                    return callback(err);
+                }
+                var contact = _.result(_.first(result.rows), 'doc');
+                // create a new patient with this patient_id
+                var patient = {
+                    name: doc.fields[patientNameField],
+                    parent: contact && contact.parent,
```

Comment:
creating a `person` without a patient is not cool. We should add least log an error, and possibly not create the patient at all (if there's no parent the reminders are probably going to get lost anyway)